scipy/stats/tests/test_stats.py

```diff
@@ -3381,31 +3381,37 @@ def test_moment_array_api(self, xp, order, axis, center):
         xp_assert_close(res, ref)
 
 
-class TestSkewKurtosis:
+class SkewKurtosisTest:
     scalar_testcase = 4.
     testcase = [1., 2., 3., 4.]
     testmathworks = [1.165, 0.6268, 0.0751, 0.3516, -0.6965]
 
+
+class TestSkew(SkewKurtosisTest):
     def test_empty_1d(self):
+        # This is not essential behavior to maintain w/ array API
```

Comment:
perhaps more clear to `@skip_xp_backends(np_only=True)`, although I don't have a strong opinion. Right now, not parameterising with `xp` very clearly means 'this is just for NumPy'. In the future, when we potentially have the majority of tests converted, that might not be so clear. But happy with this for now, can be left for the future.

Reply:
I will consider talking this on to my next PR. But honestly this test can probably just be removed. We've talked about silencing this warning throughout stats.